app/controllers/boards_controller.rb

```diff
@@ -1,4 +1,6 @@
 class BoardsController < ApplicationController
+  before_filter :authorize
```

Comment:
wiekszosc widoków bedzie wymagało authorize ( a dokladnie ) authenticate a nie authorize ;)
wiec zrób tak ze wywołaj before_action ( before_filter to to samo ale to stara konwencja z rails 3 ) w application controllerze a gdzie nie bedzie wymagany auth to robisz `skip_before_action :authenticate`